gott.go

```diff
@@ -50,57 +50,44 @@ func (c config) Flatten() map[string]string {
 	return results
 }
 
-// return a copy of self with path promoted to top level data
-func (c config) Promote(path []string) config {
-	result := config{}
-	mergo.Merge(&result, c)
-
-	if err := mergo.Merge(&result, c.Narrow(path)); err != nil {
-		panic(err)
-	}
-
-	return result
-}
-
-func (c config) Narrow(path []string) config {
-	// "copy"
-	result := config{}
-	mergo.Merge(&result, c)
-
-	var dig map[string]interface{} = c
-	for _, key := range path {
-		dig = dig[key].(map[string]interface{})
-	}
-
-	return dig
-}
```

Comment:
this is the part of the program i found most confusing, because it felt very not-go. merging and digging for specific keys didn't make much sense to me—i'm not sure if that's a personal bias or an overall complexity thing.

i approach was just to rip it out, since i couldn't come up with a good way to make them more obvious